spec/suites/SpecHelper.js

```diff
@@ -79,5 +79,33 @@ var touchEventType = L.Browser.touchNative ? 'touch' : 'pointer'; // eslint-disa
 // Note: this override is needed to workaround prosthetic-hand fail,
 //       see https://github.com/Leaflet/prosthetic-hand/issues/14
 
+function createContainer() {
+	var container = document.createElement("div");
+	container.style.position = 'absolute';
+	container.style.top = '0px';
+	container.style.left = '0px';
+	container.style.height = '400px';
+	container.style.width = '400px';
+	container.style.opacity = '0.4';

```

Comment:
Any particular reason to have 0.4 opacity here?

Reply:
Yes, the map is displayed always in the top left corner and hides the passed tests behinde it. With opacity it is possible to see while running if the tests are worked or failed without waiting until all tests passed or there is a slot without a map

Reply:
![grafik](https://user-images.githubusercontent.com/19800037/145616928-663966a2-9770-42da-a509-de14bbea7cbc.png)

Reply:
Can we just put the test results list over the map, e.g. with `position: relative` and a higher `z-index`?

Reply:
No because of the click events by coords

Reply:
Maybe position it to the right rather than left then?

Reply:
While merging the newest main branch into it, I saw again the advantages of opacity 0.4 . @mourner should I revert it back or can we keep it? Let us decide that and then merge it. It doesn't make fun to update this PR always after changes in the main 😄

Reply:
Let's keep, not a strong opinion on this one, and we'll be able to quickly update it in the future now.